ptsprojects/zephyr/gap.py

```diff
@@ -71,6 +71,10 @@ class CHAR:
 
 iut_device_name = 'Tester'
 iut_manufacturer_data = 'ABCD'
+iut_appearance = '1111'
```

Comment:
hmm, this appearance could be set to something more meaningful. Check Bluetooth assigned numbers